src/TestAppsCircles.WPF/packages.config

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I suggest you to use PackageReference

Reply:
bee832887e1d81d45b144c8f5ecfbb7c0f109ec7 is ok